x-pack/libbeat/docker-compose.yml

```diff
@@ -27,3 +28,21 @@ services:
     extends:
       file: ${ES_BEATS}/testing/environments/${TESTING_ENVIRONMENT}.yml
       service: elasticsearch
+    healthcheck:
+      test: ["CMD", "curl", "-u", "elastic:changeme", "-f", "http://localhost:9200"]
+    environment:
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - "network.host="
+      - "transport.host=127.0.0.1"
+      - "http.host=0.0.0.0"
+      - "xpack.security.enabled=true"
+      - "xpack.license.self_generated.type=trial"
```

Comment:
@exekias Interesting, didn't know such a config exist. We should probably also use this for the ML tests as there we trigger it through the API.

Reply:
yes, that would still enable it for all tests in that scenario, I'm wondering if that's what we want, or if it has some side effects